tools/README.md

```diff
@@ -108,3 +108,48 @@ def allocate_memory():
 
 max_memory_usage: float = mmc.memory_data[MemoryType.SYSTEM]
 ```
+
+## Visualization of Weight Compression results
+
+The [visualize_compression_results.py](visualize_compression_results.py) script is a useful tool for visualizing the results of weight compression.
+The result of the script is a .md file with a table:
+
+| mode                                      | %int4   | %int8   | lora<br>rank   | average<br>relative<br>error   | compression<br>rate   |
+|:------------------------------------------|:--------|:--------|:---------------|:-------------------------------|:----------------------|
+| fp32                                      | 0%      | 0%      |                | 0.0%                           | 1.0x                  |
+| int8                                      | 0%      | 100%    |                | 1.0%                           | 4.0x                  |
+| int4 + scale estimation + lora correction | 100%    | 0%      | 256.0          | 3.9%                           | 6.0x                  |
+| int4 + scale estimation                   | 40%     | 60%     |                | 4.1%                           | 4.8x                  |
+| int4 + scale estimation                   | 60%     | 40%     |                | 4.3%                           | 5.4x                  |
+| int4 + scale estimation + lora correction | 100%    | 0%      | 128.0          | 4.6%                           | 6.5x                  |
+| int4 + scale estimation                   | 80%     | 20%     |                | 5.7%                           | 6.1x                  |
+| int4 + scale estimation + lora correction | 100%    | 0%      | 8.0            | 5.8%                           | 7.1x                  |
+| int4 + scale estimation + gptq            | 100%    | 0%      |                | 6.1%                           | 7.1x                  |
+| int4 + scale estimation                   | 100%    | 0%      |                | 7.5%                           | 7.1x                  |
+| int4                                      | 100%    | 0%      |                | 11.9%                          | 7.1x                  |
+
+Also it plots a trade-off between accuracy and footprint by processing a CSV file in a specific format.
+The resulting images are employed for [the relevant section](../docs/usage/post_training_compression/weights_compression/Usage.md#accuracyfootprint-trade-off) in the Weight Compression documentation:
```

Comment:
```suggestion
The resulting images are employed for [the relevant section](/docs/usage/post_training_compression/weights_compression/Usage.md#accuracyfootprint-trade-off) in the Weight Compression documentation:
```
 it can be link relative to root, instead of this file, in some case it be easy to support

Reply:
Relative to root works for Github, but VSCode struggles to visualize now 
![image](https://github.com/user-attachments/assets/9910cb34-17cc-440a-8aa4-38e28d5337a5)